test/client.test.js

```diff
@@ -379,6 +378,20 @@ tap.test('createClient', t => {
     }
   })
 
+  t.test('url array is correctly assigned', async t => {
+    getPort().then(function (unusedPortNumber) {
+      const client = ldap.createClient({
+        url: [
+          `ldap://0.0.0.0:${unusedPortNumber}`,
+          `ldap://0.0.0.1:${unusedPortNumber}`
```

Comment:
It looks like the tests are failing because the server can't connect to 0.0.0.0
```suggestion
          `ldap://127.0.0.1:${unusedPortNumber}`,
          `ldap://127.0.0.2:${unusedPortNumber}`
```

Reply:
Weird that sometimes it succeeds sometimes it fails. I checked for active handles after the test is over and sometimes the sockets stay in paused mode (hence the fail, because the test didn't terminate itself).